src/style.css

```diff
@@ -1,3 +1,82 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  padding-left: 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+}
+
+.menu {
+  justify-content: center;
+  display: flex;
+  margin-right: 50px;
+}
+
+.menu__link {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  height: 60px;
+  align-items: center;
+  display: flex;
+}
+
+.menu__link:hover {
+  color: #00acdc;
+}
+
+.menu__item {
+  list-style-type: none;
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-align: center;
+  margin-left: 20px;
+}
+
+.menu__item:first-child {
+  margin-left: 0;
+}
+
+.menu__list {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0;
+  margin: 0;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: '';
+  left: 0;
+  position: absolute;
+  width: 100%;
+  bottom: 0;
+  height: 4px;
+  background: #00acdc;
```

Comment:
Let's use variables for repeating properties.

![image](https://github.com/mate-academy/layout_moyo-header/assets/54860641/09d41120-1cd7-4735-a0f3-7bc0be98156c)